manifests/rt/sno-realtime.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: "master"
```

Comment:
You can make this yaml file a jinja2 file with the role being a variable and render the desired yaml file. Thus you can dynamically replace the role (based on SNO vs. non SNO). This will save you 1 file being commit to the repo.